packages/components/pie-radio-group/src/index.ts

```diff
@@ -11,10 +25,133 @@ const componentSelector = 'pie-radio-group';
 
 /**
  * @tagname pie-radio-group
+ * @slot - Default slot
+ * @slot label - The label slot
+ * @event {CustomEvent} change - when one of the radios state is changed.
  */
-export class PieRadioGroup extends LitElement implements RadioGroupProps {
+export class PieRadioGroup extends FormControlMixin(RtlMixin(LitElement)) implements RadioGroupProps {
+    @state()
+        hasLabel = false;
+
+    @property({ type: String })
+    public name?: RadioGroupProps['name'];
+
+    @property({ type: String })
+    public value = defaultProps.value;
+
+    @property({ type: Boolean })
+    public isInline = defaultProps.isInline;
+
+    @property({ type: Boolean, reflect: true })
+    public disabled = defaultProps.disabled;
+
+    @queryAssignedElements({ selector: 'pie-radio' })
+        _slottedChildren: Array<HTMLInputElement> | undefined;
```

Comment:
Perhaps same here? Unsure if its just github 😄

Reply:
I think the linter thinks that this is correct whitespace when you're not using the `private` access modifier, I reckon we should add it

Reply:
https://github.com/justeattakeaway/pie/pull/2006#discussion_r1816195668